### PR TITLE
MARKET-5422: Merge to Main

### PR DESCRIPTION
**Pull request by co185063**
fixed docker file to include the migrate command and removed the secret key that doesn't need to be exposed

## Description
Please include a description of what this pull request does or what it is trying to solve. This pull requests fixes XYZ or implements ABC.

## Screenshots/Screencasts
If applicable.

## Testing
Please outline how to test this fix or feature. Include all steps.

1. Login to XYZ
2. Click on Button
3. Confirm ABC.

## Ticket
Ticket Number

### CODE REVIEW

File: code/BurgersUnlimited/settings.py [hunk 1]
```diff
@@ -22,11 +22,11 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '+=ey4af%pw@67a!knl6t44e)$ms=6uq7x)btzilz4yl$2c7!2v'
+SECRET_KEY = ''
```
Comment (cb185224) [inline code/BurgersUnlimited/settings.py:25]: No need to do this.

File: code/BurgersUnlimited/settings.py [hunk 2]
```diff
@@ -22,11 +22,11 @@
...
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-mimetypes.add_type("application/javascript", ".js", True)
+DEBUG = False
```
Comment (cb185224) [inline code/BurgersUnlimited/settings.py:28]: Let's leave this in until the fix is confirmed.

**Review by cb185224 — CHANGES_REQUESTED** [review]

File: code/BurgersUnlimited/settings.py (hunk 1 above)
Comment (co185063) [reply code/BurgersUnlimited/settings.py]: I've added this back in.

File: code/BurgersUnlimited/settings.py (hunk 2 above)
Comment (co185063) [reply code/BurgersUnlimited/settings.py]: I have added it back in

**Review by cb185224 — APPROVED** [review]

**Pull request merged**